run.py

```diff
@@ -454,6 +458,7 @@ def fit(self):
 	parser.add_argument('-gcn_drop',	dest='dropout', 	default=0.1,  	type=float,	help='Dropout to use in GCN Layer')
 	parser.add_argument('-hid_drop',  	dest='hid_drop', 	default=0.3,  	type=float,	help='Dropout after GCN')
 	parser.add_argument('-disable_gnn_encoder', dest='disable_gnn_encoder', default=False, type=bool, help='Disables the GNN encoder layer')
+	parser.add_argument('-optimizer', dest='optimizer', default='Adam', help='Sets the Optimizer to Adagrad')
```

Comment:
```suggestion
	parser.add_argument('-optimizer', dest='optimizer', default='Adam', help='Optimizer to use')
```

Reply:
I reworked the Help sentence, to include the default value and possible values.